### ISSUE TITLE
Avoid mutating the python path

### ISSUE THREAD

**Issue by JeffreyDevloo** [openvstorage/framework#1897]
### Problem description
Looking into a conflict with openvswitch packages, I found at the we mutate the python path very often in our code to make everything work as expected
However this is the subawesome way of tackling this issue.

I propose to use https://docs.python.org/2.7/library/site.html instead

Script to include in postinstall
```
# find directory
SITEDIR=$(python -m site --user-site)

# create if it doesn't exist
mkdir -p "$SITEDIR"

# create new .pth file with our path
echo "$HOME/foo/bar" > "$SITEDIR/somelib.pth"
```

**Comment by JeffreyDevloo** [comment]
Would be fixed by: 
- https://github.com/openvstorage/framework/pull/1899
- https://github.com/openvstorage/framework-alba-plugin/pull/542
- https://github.com/openvstorage/framework-iscsi-plugin/pull/39

**Comment by sjovdnbos** [comment]
- Framework:  fixed by #1899, package information: openvstorage-2.12.0-dev.1517313757.e96a07f
- Framework alba-plugin: https://github.com/openvstorage/framework-alba-plugin/pull/542 , package information:  openvstorage-backend-1.12.0-dev.1517317588.d0d841d
- Framework iscsi-plugin: https://github.com/openvstorage/framework-iscsi-plugin/pull/39, package information: openvstorage-iscsi-plugin-0.4.0-dev.1517317694.a997306

**Comment by JeffreyDevloo** [comment]
Released in https://github.com/openvstorage/framework/releases/tag/2.12.0
Packaged in openvstorage-core_2.12.0-1_amd64.deb
